src/main/cli/algo.py

```diff
@@ -1,15 +1,22 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
+import sys
+import logging
+
+sys.path.append('.')
```

Comment:
Did you try to run it right from the src/main/cli path by calling `python algo.py <args>`?

Reply:
Yes, it works on my laptop (I have macOS). I tried to run it from:
- the `root` project
- the `src` folder
- the `main` folder
- the `cli` folder